### DISCUSSION TITLE
Slideshow play videos

### DISCUSSION THREAD

**Discussion by DeltaTango69** [photoprism/photoprism#4031] [Q&A]
Hello,

I am very new to Photoprism and trying to understand things. I have at least managed to get things working with network drives under Windows using Docker Compose, which was not so easy as a newcomer. I can't quite get to grips with the operation in moment. I would like to play images and videos in the correct chronological order, i.e. from old to new. In the overview, however, everything is sorted from new to old. I can reverse the order, but how do I get to the end with almost 60000 videos and pictures? In addition, the videos are not played but only a still image is shown. Are there any settings that I may have overlooked?

**Comment by graciousgrey** [comment]
> In the overview, however, everything is sorted from new to old. I can reverse the order, but how do I get to the end with almost 60000 videos and pictures? 

You can limit the number of photos shown by using the search. If you want to see photos from oldest to newest but only from a specific time on you can sort by oldest and use the search filter "after".

https://docs.photoprism.app/user-guide/search/filters/#filter-reference

>  In addition, the videos are not played but only a still image is shown. Are there any settings that I may have overlooked?

When you enter a video in  a slideshow you need to click the play button. It is located under the photo next to the photo title. Improvements for this are planned.

- https://github.com/photoprism/photoprism/issues/2868

**Reply by uc0ne** [reply]
I‘m sorry to get this older thread up, but despite #2868 my videos in the slideshow are starting only at the beginning of a slideshow. Upcoming videos aren’t playing. There’s just a thumbnail with a manual play button. 

I’m using 250707 release.

**Reply by lastzero** [reply]
Do you use Safari?

**Reply by uc0ne** [reply]
Yes on an iPad

**Reply by graciousgrey** [reply]
Unfortunately videos do only auto-play in safari if they are muted due to some safari policies. As far as I remember this can be turned off in safari on desktop devices but not on mobile.

**Reply by uc0ne** [reply]
Thank you very much for your support!
